.github/workflows/release-commit.yml

```diff
@@ -28,4 +32,4 @@ jobs:
         run: ${{ inputs.build }}
 
       - name: Publish
-        run: pnpm dlx pkg-pr-new@0.0 publish --pnpm ${{ inputs.compact && '--compact' || '' }} ${{ inputs.packages }}
+        run: pnpm dlx pkg-pr-new@0.0 publish --pnpm ${{ inputs.compact && '--compact' || '' }} ${{ inputs.packages }} ${{ inputs.packageManager && `--packageManager=${{ inputs.packageManager }}` || '' }}
```

Comment:
<code>`</code> is not a valid syntax I guess

Reply:
Should we remove `--pnpm`?

Reply:
> Should we remove `--pnpm`?

No, `--pnpm` is for executing `pnpm pack`, which converts the catalog protocol into actual dependency versions.